src/python_testing/TC_AVSUM_2_4.py

```diff
@@ -203,7 +203,7 @@ async def test_TC_AVSUM_2_4(self):
             if canbemadebusy:
                 self.step(18)
                 # Busy response check
-                if not self.is_ci:
```

Comment:
I never use 'is_ci', what is this for? and under what circumstance we should use this one instead of 'is_pics_sdk_ci_only'